GRPCServiceA/pkg/entities/requests.go

```diff
@@ -0,0 +1,14 @@
+package entities
+
+type GetUserReq struct {
+	Id string
+}
+
+type CreateUserReq struct {
```

Comment:
this should be just the domain entity, by default is named as the main entity or entities in the domain, "user" UserReq refers to a struct with JSON annotations uses in transport or in infra layer